nixos/tests/incron.nix

```diff
@@ -0,0 +1,52 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
+
+{
+  name = "incron";
+  meta = [ lib.maintainers.aanderse ];
```

Comment:
This should be `meta.maintainers = lib.maintainers.aanderse;` :)

Reply:
That is what I originally thought too, but then I saw this...

https://github.com/NixOS/nixpkgs/commit/2380f6a4faa57c6b91fed26c496e1c8ca5d91982#diff-e8c90503ae54b74ba0326db099bc68ff

What am I missing?

Reply:
Hmm not sure, but there's other files in `nixpkgs/nixos/tests` using the `meta.maintainers` way.